sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -318,7 +318,7 @@ public async Task AnalyzeOperationWithPagination()
             Assert.IsFalse(operation.HasValue);
 
             Assert.ThrowsAsync<InvalidOperationException>(async () => await Task.Run(() => operation.Value));
-            Assert.Throws<InvalidOperationException>(() => operation.GetValues());
```

Comment:
our tests were only validating the `GetValues` function which was already doing the right thing.
By using the `GetValuesAsync` function here, our Test Framework is calling both `GetValues` and `GetValuesAsync`